edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -88,6 +92,36 @@ impl DockerModuleRuntime {
         self
     }
 
+    pub fn with_network_configuration(mut self, network_configuration: MobyNetwork) -> Self {
+        self.network_id = Some(network_configuration.name().to_string());
+        if let MobyNetwork::Network(network) = network_configuration {
+            self.ipv6 = network.ipv6().unwrap_or_else(|| &false).to_owned();
```

Comment:
`self.ipv6 = network.ipv6().copied().unwrap_or_default();`

Reply:
After you make `Network::ipv6` return `Option<bool>`, the `.copied()` is unnecessary, so just `network.ipv6().unwrap_or_default()`